docs/pages/overview.mdx

```diff
@@ -3,14 +3,25 @@ title: Overview
 hideTOC: true
 ---
 
+import { TerminalBrowserIcon, Rocket01Icon } from '@expo/styleguide-icons';
+import { HandWaveIcon } from '~/ui/components/CustomIcons/HandWaveIcon';
+
 Expo is an [open-source framework](https://github.com/expo/expo/) for apps that run natively on Android, iOS, and the web. Expo brings together the best of mobile and the web and enables many important features for building and scaling an app.
 
-- **Develop**: Developing an app requires writing code in JavaScript/TypeScript. When you create a new project, Expo provides a standard [project structure](/develop/project-structure) which you can customize as per your needs. It also provides [development builds](/develop/development-builds/introduction) that help your team to iterate quickly to achieve web-like iteration speeds and safely.
+### <span className="inline-flex items-center gap-2"><HandWaveIcon className="icon-md" />Get Started</span>
+
+Learn about prerequisites, and how to quickstart an universal project with Expo.
```

Comment:
```suggestion
Learn about prerequisites and how to quickly start a universal project with Expo.
```

- Let's fix the grammatical error before universal
- Also, we don't need to use comma here